nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -995,6 +1012,22 @@ def quantize_input(self, inputs, wrapper, **kwargs):
         inputs = self.quantize(inputs, module.input_scale, module.input_qmin, module.input_qmax)
         return inputs
 
+    def load_calibration_config(self, calibration_config):
+        for name, module in self.bound_model.named_modules():
+            if name not in calibration_config:
+                if hasattr(module, 'weight_bits') or hasattr(module, 'output_bits'):
+                    logger.warning(f"Can not find module {name}'s parameter in input config.")
+                continue
+            if hasattr(module, 'weight_bits'):
+                assert calibration_config[name]['weight_bits'] == module.weight_bits, f"weight bits of module {name} fail to match"
+            if hasattr(module, 'input_bits'):
+                assert calibration_config[name]['input_bits'] == module.input_bits, f"input bits of module {name} fail to match"
+                module.input_scale = float(calibration_config[name]['tracked_max_input'] / module.input_qmax)
```

Comment:
`input_scale` is` torch.nn.Parameter` while the right operand is `float`

Reply:
exactly==, have modified.